service/src/main/java/uk/gov/hmcts/reform/fpl/service/MessageJudgeService.java

```diff
@@ -298,65 +44,7 @@ private String buildMessageHistory(String message, String history, String sender
         return join("\n \n", history, formattedLatestMessage);
     }
 
-    private DynamicList rebuildJudicialMessageDynamicList(CaseData caseData, UUID selectedC2Id) {
```

Comment:
Most of this deleted stuff has been added to the new SendNewMessageJudgeService right?

Reply:
not all of them, some of them are only used in reply message flow. e.g. updateJudicialMessages, replyToJudicialMessage, rebuildJudicialMessageDynamicList